src/components/common/Job/jobStyles.ts

```diff
@@ -0,0 +1,112 @@
+import { makeStyles, createStyles, Theme } from '@material-ui/core/styles';
+
+const styles = (theme: Theme) => createStyles({
+    root: {
+        maxWidth: 345,
+    },
+    tag: {
+        width: "4ch",
+        height: "1.5rem",
+        textAlign: "center",
+        color: "white",
+        fontFamily: 'unset',
+        borderRadius: "5px"
+    },
+    job: {
+        boxShadow: "14px 0px 8px 2px #bbbbbb, -14px 0px 8px 2px #bbbbbb",
+        marginBottom: "30px",
+        display: "flex",
+        flexDirection: "row",
+        height: '14vh',
+        minHeight: '120px',
+        width: "100%",
+        minWidth: '1220px'
+    },
+    jobHeader: {
+        display: "flex",
+        flexDirection: "column",
+        padding: "20px",
+        width: "19vw"
+    },
+    jobTitle: {
+        fontWeight: "bold",
+        fontSize: "24px",
+        marginBottom: "10px",
+        color: "#595959"
```

Comment:
Take `color` from `assets`. As they are the same color throught the project 😃 

I left jpg there, and also you can see in comments of the import place.

Reply:
Since this is not my code, and I am in a but of a hurry, I am going to leave it like this for now (regarding all css)